crates/bevy_ecs/src/world/mod.rs

```diff
@@ -2206,6 +2206,9 @@ impl World {
 
     /// Applies any commands in the world's internal [`CommandQueue`].
     /// This does not apply commands from any systems, only those stored in the world.
+    ///
+    /// This will panic if any of the queued commands are [`spawn`](Commands::spawn),
+    /// instead use [`flush`](Self::flush) if there is a chance of this.
```

Comment:
This is a comma splice. You can fix it by making 2 separate sentences or with a semicolon and comma.


    /// This will panic if any of the queued commands are [`spawn`](Commands::spawn).
    /// If this is possible, you should instead use [`flush`](Self::flush).

    /// This will panic if any of the queued commands are [`spawn`](Commands::spawn);
    /// in which case, use [`flush`](Self::flush).